frontend/src/models/cohortsModel.ts

```diff
@@ -54,6 +54,7 @@ export function processCohort(cohort: CohortType): CohortType {
                         ? {
                               ...group,
                               values: (group.values as AnyCohortCriteriaType[]).map((c) => processCohortCriteria(c)),
+                              sort_key: generateUUID(),
```

Comment:
Please rebase this branch (or update the branch) from the `master` branch. We're now using `uuidv4()`. Thanks!